### PR TITLE
updated Portuguese (Brazilian) transalations (#900)

### PR DESCRIPTION
**Pull request by MarcosEllys**
## Description
Updated some strings in pt_BR language.

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist
- [x] I verified that my code applies to the guidelines (`composer kimai:code-check`)
- [x] I updated the documentation (see [here](https://github.com/kimai/www.kimai.org/tree/master/_documentation))
- [x] I agree that this code is used in Kimai and will be published under the [MIT license](https://github.com/kevinpapst/kimai2/blob/master/LICENSE)

## Issue

#900 - Updated Portuguese (Brazilian) translation

### CODE REVIEW

File: translations/messages.pt_BR.xliff [hunk 1]
```diff
@@ -837,6 +928,22 @@
...
             </trans-unit>
+
+
+            <trans-unit id="export.clear_all">
+                <source>export.clear_all</source>
+                <target>Marcar todos os registros mostrados como aberto?</target>
+            </trans-unit>
+            <trans-unit id="export.mark_all">
+                <source>export.mark_all</source>
+                <target>Mark all shown records as cleared?</target>
```
Comment (kevinpapst) [inline translations/messages.pt_BR.xliff]: this line needs to be removed, probably the build will work then

**Comment by kevinpapst** [comment]
Thanks for your support 👍

File: translations/messages.pt_BR.xliff (hunk 1 above)
Comment (MarcosEllys) [reply translations/messages.pt_BR.xliff]: My mistake, fixed now

File: translations/system-configuration.pt_BR.xliff [hunk 2]
```diff
@@ -0,0 +1,99 @@
+<?xml version="1.0" encoding="utf-8"?>
+<xliff version="1.2" xmlns="urn:oasis:names:tc:xliff:document:1.2">
+    <file source-language="en" target-language="en" datatype="plaintext" original="system-configuration.en.xliff">
```
Comment (kevinpapst) [inline translations/system-configuration.pt_BR.xliff]: target-language="pt-BR"

**Comment by kevinpapst** [comment]
Can you compare messages.en.xliff and messages.pt_BR.xliff ? 
There are some keys missing: `menu.system`, `menu.export`, `menu.plugin`, `menu.system_configuration`, `menu.tags`, `label.tag`, `label.tags`, `label.daterange`, `label.color`, `label.login.initial_view`, `label.timesheet.daily_stats`, `label.version`, `label.required_version`, `stats.yourWorkingHours` and some buttons

**Comment by MarcosEllys** [comment]
> Can you compare messages.en.xliff and messages.pt_BR.xliff ?
> There are some keys missing: `menu.system`, `menu.export`, `menu.plugin`, `menu.system_configuration`, `menu.tags`, `label.tag`, `label.tags`, `label.daterange`, `label.color`, `label.login.initial_view`, `label.timesheet.daily_stats`, `label.version`, `label.required_version`, `stats.yourWorkingHours` and some buttons

Where i find the buttons?

**Comment by kevinpapst** [comment]
also in messages file. I think `buttons.csv` and three others. They are used in the export view.

**Comment by MarcosEllys** [comment]
> also in messages file. I think `buttons.csv` and three others. They are used in the export view.

Done in this commit c0cd9f57378be18edd6f5376abc999450b94aaf9

**Pull request merged**